Code/Editor/AnimationContext.cpp

```diff
@@ -671,34 +669,6 @@ void CAnimationContext::OnPostRender()
     }
 }
 
-//////////////////////////////////////////////////////////////////////////
-void CAnimationContext::UpdateAnimatedLights()
-{
-    bool bLightAnimationSetActive = m_pSequence && (m_pSequence->GetFlags() & IAnimSequence::eSeqFlags_LightAnimationSet);
-    if (bLightAnimationSetActive == false)
-    {
-        return;
-    }
-
-    std::vector<CBaseObject*> entityObjects;
-    GetIEditor()->GetObjectManager()->FindObjectsOfType(&CEntityObject::staticMetaObject, entityObjects);
-    std::for_each(std::begin(entityObjects), std::end(entityObjects),
-        [this](CBaseObject* pBaseObject)
-        {
-            CEntityObject* pEntityObject = static_cast<CEntityObject*>(pBaseObject);
-            bool bLight = pEntityObject && pEntityObject->GetEntityClass().compare("Light") == 0;
-            if (bLight)
-            {
-                bool bTimeScrubbing = pEntityObject->GetEntityPropertyBool("bTimeScrubbingInTrackView");
```

Comment:
Have you verified whether this is supposed to be updating something for TrackView sequences?

Reply:
I can't see a way to create a CEntityObject.